### DISCUSSION TITLE
MONAI UNet = 3D?

### DISCUSSION THREAD

**Discussion by ghost** [Project-MONAI/MONAI#1523] [Q&A] [answered]
Hey,
I've seen some mentions that the UNet implemented by MONAI is 3D but could someone please confirm? I tried searching these discussions for an answer but with no luck.
I've tried to read the cited paper but can't find a full copy online through my university. When reading this snippet I couldn't discern that it was necessarily 3D:
https://books.google.co.uk/books?hl=en&lr=&id=mqOLDwAAQBAJ&oi=fnd&pg=PA369&ots=GTN4I9jsbK&sig=7aY26i523oRNGnj4TiuZnVXFmSE#v=onepage&q&f=false 
Thanks

**Comment by Nic-Ma** [comment, marked as answer]
Hi @jnc13 ,

Thanks for your interest here, there are 2 UNet implementations in MONAI, both support 2D and 3D.
1. https://github.com/Project-MONAI/MONAI/blob/master/monai/networks/nets/unet.py
2. https://github.com/Project-MONAI/MONAI/blob/master/monai/networks/nets/basic_unet.py

And there are many 3D examples in MONAI tutorials, you can search `UNet` in: https://github.com/Project-MONAI/tutorials.

Thanks.
